src/main/telemetry/crsf.c

```diff
@@ -413,13 +414,13 @@ void crsfFrameFlightMode(sbuf_t *dst)
     if (!ARMING_FLAG(ARMED) && !FLIGHT_MODE(FAILSAFE_MODE)) {
         // * - ready to arm
         // ! - arming disabled
-        // ? - GPS fix not available
-        bool isAllowedArmingWithoutFix = false;
+        // ? - GPS rescue disabled (when allowArmingWithoutFix is enabled)
```

Comment:
Can’t understand this comment.
Just delete it?

Reply:
When allowArmingWithoutFix is enabled and we do not have required sats and gps fix show ? to indicate.